src/main/pages/che-7/overview/assembly_ide-projects.adoc

```diff
@@ -0,0 +1,155 @@
+---
```

Comment:
This looks like a duplicate of `proc_creating-a-workspace-by-importing-source-code-of-your-project.adoc`

Reply:
See above. deleted in PR update (coming soon)